NEWS.md

```diff
@@ -6,11 +6,20 @@ where the formatting is also better._
 
 ## 0.2.1.99 (dev version)
 
+New features:
+
+- The new `flip` argument allows for easily flipping (swapping) the orientation
+of the x and y axes. This should work regardless of plot type, e.g.
+`plt(~Sepal.Length, data = iris, type = "density")`. (#216 @grantmcdermott)
```

Comment:
Where's the `flip` argument in this call?

Reply:
D'oh. good catch, thanks.